bitnami/external-dns/values.yaml

```diff
@@ -858,6 +858,10 @@ txtEncrypt:
   ## This ignores txtEncrypt.aesKey
   ##
   secretName: ""
+## @param txtNewFormatOnly When using the TXT registry, use only the new format for ownership records (optional)
+## ref: https://github.com/kubernetes-sigs/external-dns/blob/master/docs/registry/txt.md#record-format-options
+##
+txtNewFormatOnly: false
```

Comment:
I just noticed this section was added away from the other "txt" related parameters. Can you move it up just after the txtSuffix one?